target/linux/ath79/dts/qca9558_comfast_cf-wr650ac-v2.dts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9558_comfast_cf-wr650ac.dtsi"
+
+/ {
+	compatible = "comfast,cf-wr650ac-v2", "qca,qca9557";
```

Comment:
I think this should be qca,qca9558 instead of 9557 ...

Reply:
fixed